### ISSUE TITLE
Unable to create service with blank name

### ISSUE THREAD

**Issue by bwlng** [nystudio107/generator-craftplugin#12]
Creating a Service with a blank name on v1.1.18 causes an error (`Cannot read property 'toLowerCase' of undefined`) due to this bit on line 33 of `/app/templates/api_version_2_5/services/_Service.php`

```
<%= "_" + serviceName[index][1].toLowerCase() + serviceName[index].slice(2) %>
```

I worked around it by temporarily removing that portion of the code, but don't know what the proper fix is to contribute.

**Comment by khalwat** [comment]
Ooopsie, I'll fix it, thanks.

**Comment by khalwat** [comment]
This is fixed in 1.1.19, and I pushed it to pluginfactory.io

https://github.com/nystudio107/generator-craftplugin

**Comment by bwlng** [comment]
Thank you, works like a charm :)
